tests/console/yast2_xinetd.pm

```diff
@@ -0,0 +1,87 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use strict;
+use base "consoletest";
+use testapi;
+
+
+
+sub run() {
+    select_console 'root-console';
+
+    # check network at first
+    assert_script_run("if ! systemctl -q is-active network; then systemctl -q start network; fi");
+    # install xinetd at first
+    assert_script_run("/usr/bin/zypper -n -q in xinetd");
+
+    script_run("/sbin/yast2 xinetd; echo yast2-xinetd-status-\$? > /dev/$serialdev", 0);
+
+    # check xinetd network configuration got started
+    assert_screen 'yast2_xinetd_startup';
+
+    # enable xinetd
+    send_key 'alt-l';
+    wait_still_screen
+
+      # toggle status on at first and then off
+      send_key 'alt-s';
```

Comment:
@okurz it seems that tidy cannot handle the line with toggle... Check this and change it manually, thanks.